packages/api-fetch/src/index.js

```diff
@@ -36,6 +36,10 @@ function apiFetch( options ) {
 			headers[ 'Content-Type' ] = 'application/json';
 		}
 
+		if ( ! headers.Accept ) {
+			headers.Accept = 'application/json, */*;q=0.1';
```

Comment:
Do we _want_ the "any" fallback?

Reply:
Sorry, I thought that was resolved in the message with @youknowriad. Previously, `*/*` would be sent. Since I guess `apiFetch` isn't only intended for use with JSON APIs, I think it makes sense to use `*/*` as a fallback. Particularly since they can disable `parse` to allow them to handle their own responses as necessary, meaning that requiring JSON would no longer be true.

Reply:
>Since I guess `apiFetch` isn't only intended for use with JSON APIs

But I shall prove that it is _!_

Since the API Fetch module is a ["utility to make WordPress REST API requests"](https://github.com/WordPress/gutenberg/tree/master/packages/api-fetch) and the REST API operates by ["sending and receiving JSON"](https://developer.wordpress.org/rest-api/) we can conclude that — via transitive relation — the API Fetch module sends and receives JSON.

Reply:
I don't have any strongly held opinion here, just going off of @youknowriad's feedback.